pkg/porter/generateManifest.go

```diff
@@ -41,5 +41,13 @@ func (p *Porter) generateInternalManifest(opts BuildOptions) error {
 		}
 	}
 
+	if opts.parsedCustoms != nil {
```

Comment:
You can leave off this if check. The for loop won't do anything if the map is nil.